iset.mm

```diff
@@ -98005,6 +98021,113 @@ This definition (in terms of ` U. ` and ` ~<_ ` ) is not taken directly
+    ( cfn csize cfv cc0 cap wbr wne clt c0 cz wb sizecl nn0zd 0zd zapne syl2anc
+    wcel cn0 cr cle nn0re nn0ge0 ap0gt0 syl sizeeq0 necon3bid 3bitr3d ) ABRZACD
+    ZEFGZUJEHZEUJIGZAJHUIUJKREKRUKULLUIUJAMZNUIOUJEPQUIUJSRZUKUMLZUNUOUJTREUJUA
+    GUPUJUBUJUCUJUDQUEUIUJEAJAUFUGUH $.
+
+  $( Positive natural closure of the hash function.  (Contributed by Jim
+     Kingdon, 23-Feb-2022.) $)
+  sizenncl $p |- ( A e. Fin -> ( ( size ` A ) e. NN <-> A =/= (/) ) ) $=
+    ( cfn wcel csize cfv cn c0 wne wb cc0 wceq wa simpr nnne0 sizeeq0 necon3bid
+    adantl adantr mpbid wn 2thd necon2bi biimpa syl 2falsed cn0 wo sizecl elnn0
+    nner sylib mpjaodan ) ABCZADEZFCZUOAGHZIUNJKZUMUOLZUOUPUMUOMURUNJHZUPUOUSUM
+    UNNZQUMUSUPIUOUMUNJAGAOZPRSUAUMUQLZUOUPUQUOTUMUOUNJUTUBQVBAGKZUPTUMUQVCVAUC
+    AGUJUDUEUMUNUFCUOUQUGAUHUNUIUKUL $.
+
+  $( The empty set has size zero.  (Contributed by Jim Kingdon,
+     23-Feb-2022.) $)
+  size0 $p |- ( size ` (/) ) = 0 $=
+    ( c0 csize cfv cc0 wceq eqid cfn wcel wb 0fin sizeeq0 ax-mp mpbir ) ABCDEZA
+    AEZAFAGHNOIJAKLM $.
+
+  $( The size of a singleton.  (Contributed by Jim Kingdon, 23-Feb-2022.) $)
+  sizesng $p |- ( A e. V -> ( size ` { A } ) = 1 ) $=
+    ( wcel csn csize cfv c1 cen wbr cz 1z en2sn mpan2 cfn wb snfig ax-mp sizeen
+    wceq sylancl mpbird cfz co cn0 1nn0 sizefz1 fzsn fveq2d syl5reqr syl6eq ) A
+    BCZADZEFZGDZEFZGUKUMUOSZULUNHIZUKGJCZUQKAGBJLMUKULNCUNNCZUPUQOABPURUSKGJPQU
+    LUNRTUAURUOGSKURGGGUBUCZEFZUOGUDCVAGSUEGUFQURUTUNEGUGUHUIQUJ $.
+
+  $( A set with only one element is equinumerous to the ordinal number 1.
+     (Contributed by Jim Kingdon, 23-Feb-2022.) $)
+  sizeen1 $p |- ( A e. Fin -> ( ( size ` A ) = 1 <-> A ~~ 1o ) ) $=
+    ( cfn wcel csize cfv c1 c0 csn cen wbr c1o cvv 0ex sizesng ax-mp eqcomi a1i
+    wceq eqeq2d wb snfig sizeen mpan2 df1o2 breq2i 3bitrd ) ABCZADEZFRUHGHZDEZR
+    ZAUIIJZAKIJZUGFUJUHFUJRUGUJFGLCZUJFRMGLNOPQSUGUIBCZUKULTUNUOMGLUAOAUIUBUCUL
+    UMTUGUIKAIKUIUDPUEQUF $.
+
+  $( A function on a finite set is equinumerous to its domain.  (Contributed by
+     Jim Kingdon, 24-Feb-2022.) $)
+  sizefn $p |- ( ( F Fn A /\ A e. Fin ) -> ( size ` F ) = ( size ` A ) ) $=
+    ( wfn cfn wcel wa csize cfv wceq cen fndmeng ensymd wb fnfi sizeen sylancom
+    wbr mpbird ) BACZADEZFZBGHAGHIZBAJQZUAABADBKLSTBDEUBUCMABNBAOPR $.
+
+  $( The value of the size function on a finite 1-based sequence.  (Contributed
+     by Jim Kingdon, 24-Feb-2022.) $)
+  fseq1size $p |- ( ( N e. NN0 /\ F Fn ( 1 ... N ) ) -> ( size ` F ) = N ) $=
+    ( cn0 wcel c1 cfz co wfn csize cfv cfn wceq simpr simpl nn0zd fzfigd sizefn
+    wa 1zzd syl2anc sizefz1 adantr eqtrd ) BCDZAEBFGZHZRZAIJZUEIJZBUGUFUEKDUHUI
+    LUDUFMUGEBUGSUGBUDUFNOPUEAQTUDUIBLUFBUAUBUC $.
+
+  ${
+    $d A n z $.  $d B n $.  $d G n z $.
+    omgadd.1 $e |- G = frec ( ( x e. ZZ |-> ( x + 1 ) ) , 0 ) $.
+    $( Mapping ordinal addition to integer addition.  (Contributed by Jim
+       Kingdon, 24-Feb-2022.) $)
```

Comment:
Maybe add as a second sentence: "G is a morphism from finite ordinals with ordinal addition to nonnegative integers with integer addition."
The important thing is that "morphism" appears, since a person looking for that kind of result would probably search for the keyword "morphism".

Reply:
Whether that kind of wording is helpful or off-putting probably depends on how much category theory the reader has seen. And I'm not sure how good morphism would be as a search term - set.mm already has 1367 occurrences.

I guess at least for now I'll stick with the terser wording.